st2client/st2client/commands/action.py

```diff
@@ -1022,7 +1022,7 @@ def __init__(self, description, app, subparsers, parent_parser=None):
                                                            add_help=True)
 
 
-POSSIBLE_ACTION_STATUS_VALUES = ('succeeded', 'running', 'scheduled', 'failed', 'canceling',
+POSSIBLE_ACTION_STATUS_VALUES = ('succeeded', 'running', 'scheduled', 'paused', 'failed', 'canceling',
```

Comment:
This needs to be fixed too, minor linting issue: 
```
action.py:1025:101: E501 line too long (102 > 100 characters)
```
from https://travis-ci.org/StackStorm/st2/jobs/354356559#L673